prototype2/common/EFUArgs.cpp

```diff
@@ -66,23 +82,33 @@ EFUArgs::EFUArgs() {
                   "Terminate after timeout seconds")
       ->group("EFU Options")
       ->set_default_val("4294967295"); // 0xffffffffU
-  WriteConfigOption = CLIParser.add_option("--write_config", ConfigFileName, "Write CLI options with default values to config file.")->group("EFU Options")->configurable(false);
-  ReadConfigOption = CLIParser.set_config("--read_config", "", "Read CLI options from config file.", false)->group("EFU Options")->excludes(WriteConfigOption);
+  WriteConfigOption =
```

Comment:
Personally I dislike the excessive line breaking type of formatting. To me the original is more readable

Reply:
I agree, but this is the doing of clang-format.

Reply:
well, where appropriate I use short blocks of
// clang-format off
// clang-format on